src/main/java/com/google/api/codegen/transformer/StaticLangResourceObjectTransformer.java

```diff
@@ -108,12 +108,14 @@ public RequestObjectParamView generateRequestObjectParam(
     if (!isRequired) {
       param.optionalDefault(namer.getOptionalFieldDefaultValue(fieldConfig, context));
     }
-    List<FieldCopyView> fieldCopyViews = new ArrayList<>();
+    List<StaticMemberView> fieldCopyViews = new ArrayList<>();
```

Comment:
The name implies that this refers to "static members," which doesn't quite seem right.

Also the variable name doesn't match the type now.

Reply:
I renamed the StaticMemberView class to StaticLangMemberView class, and also renamed that local var.